### ISSUE TITLE
Typos in wiki "How to Version Your Service > Existing Services"

### ISSUE THREAD

**Issue by icnocop** [microsoft/aspnet-api-versioning#507]
Hi.

I found two typos in the wiki page [Existing Services](https://github.com/Microsoft/aspnet-api-versioning/wiki/Existing-Services), both in the **second** set of code examples.

1. In the `ASP.NET WEB API` sample, `configurtion` should maybe be `config`, to match the preceding example, or spelled `configuration`.
2. There's an extra closing parenthesis after `AssumeDefaultVersionWhenUnspecified = true` in both examples.

Thank you.

**Comment by commonsensesoftware** [comment]
Good eye! Thanks. Fixed them.

I'm not really a fan of abbreviations nor contractions; especially, in the context of documentation. In this particular case, I had to weigh using `config` as a contraction or having the line wrap using `configuration`. I felt the exceptional contraction was better than a line wrap in this case. I'm a strong believer that code should read like English, so I don't mind typing a few extra characters to bring clarity. ;)
